Sources/Basics/HTTPClient/HTTPClientHeaders.swift

```diff
@@ -0,0 +1,93 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2020-2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+public struct HTTPClientHeaders: Sendable {
```

Comment:
Moved out from `LegacyHTTPClient.swift` as a type reusable in both `LegacyHTTPClient` and `HTTPClient`.